.github/workflows/build-notebooks-TEMPLATE.yaml

```diff
@@ -114,6 +115,26 @@ jobs:
           systemctl --user start homebrew.podman.service
           echo "PODMAN_SOCK=/run/user/${UID}/podman/podman.sock" >> $GITHUB_ENV
 
+      # only one db can be downloaded in one call https://github.com/aquasecurity/trivy/issues/3616
+      - name: Pre-pull Trivy vulnerabilities DB
+        if: ${{ contains(fromJson(inputs.github).event.pull_request.labels.*.name, 'trivy-scan') }}
```

Comment:
Instead of having a dedicated step for this code, can it be run as part of the `Run Trivy vulnerability scanner` step? Besides PR+Label, the Trivy scan also runs through the `schedule` trigger.

Reply:
when there are long-ish running tests that make the trivy run even later in the job, it fails even more

Reply:
fixed